packages/sync/src/class-replicastore.php

```diff
@@ -884,6 +884,12 @@ public function get_terms( $taxonomy ) {
 	 * @return \WP_Term|\WP_Error Term object on success, \WP_Error object on failure.
 	 */
 	public function get_term( $taxonomy, $term_id, $is_term_id = true ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Comment:
You're making the variable be used now.
```suggestion
	public function get_term( $taxonomy, $term_id, $is_term_id = true ) {
```

Reply:
Always good to cleanup phpcs:ignores